.gitignore

```diff
@@ -37,4 +38,5 @@ node/package-lock.json
 ruby/Gemfile.lock
 flask/test.py
 react/app.yaml
-flask/app.yaml
\ No newline at end of file
+flask/app.yaml
```

Comment:
This could also just be `**/app.yaml` if we expect to add more backend other than express and springboot 🤷